jupyterlab_bigquery/src/index.ts

```diff
@@ -46,14 +49,17 @@ async function activate(
     notebookTrack: notebookTrack,
   };
   const listProjectsService = new ListProjectsService();
+  const bigQueryService = new BigQueryService(
+    new ServerProxyTransportService()
+  );
   manager.launchWidget(
     ListItemsWidget,
     'left',
     'ListItemWidget',
     (widget: ReduxReactWidget) => {
       widget.addClass(ICONS.bigQuery);
     },
-    [listProjectsService, context],
+    [listProjectsService, bigQueryService, context],
```

Comment:
This is my first time seeing how this WidgetManager does its thing to instantiate a component. It's a little obtuse for me at first glance and it doesn't necessary allow for much type-safety, but there's no sense in making a major refactoring at this moment. We can discuss this as we look to migrate the other areas that can be switched over to the proxy transport.

Reply:
We can definitely discuss.